core/src/main/resources/hudson/model/CauseAction/summary.jelly

```diff
@@ -25,13 +25,16 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <t:summary icon="orange-square.png">
+      ${%Caused by}
```

Comment:
needed to add something to go next to the icon to make it looks okay,

any better wording or design thoughts?

![image](https://user-images.githubusercontent.com/21194782/110852956-ec926380-82aa-11eb-84ed-b404b1419038.png)

![image](https://user-images.githubusercontent.com/21194782/110853040-08960500-82ab-11eb-9c4e-312cdf352ba9.png)

Reply:
Maybe a more explicit `Build started by` or `Build caused by`? `Caused by` seems fine as well

Not sure whats a good wording for it is.

Personally I'm not a fan of the icon but that's outside the scope of this PR and with the new icons coming in this might be updated at some point.

Reply:
Looks fine to me.